src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx

```diff
@@ -1,12 +1,13 @@
 <!-- TODO in-app validate the library here. What should this be for things other than React-->
+
 ```js
-import { withInAppMessaging } from '@aws-amplify/ui-react';
+import { withInAppMessaging } from '@aws-amplify/ui-react-notifications';
 
 import '@aws-amplify/ui-react/styles.css';
 
-const App = () => (
-  {/* Your application */}
-);
+const App = () => ({
+  /* Your application */
+});
```

Comment:
Is this intentional? Returning an object from a React component is invalid

Reply:
This section describe how users can wrap their App with `withInAppMessaging`.
https://pr-5508.d39agkzf0cwtdz.amplifyapp.com/lib/in-app-messaging/integrate-your-application/q/platform/js/#integrate-in-app-messaging
The next section has a more detailed example of the content inside App component.

Reply:
Updated to
```
const App = () => (
  <>In-App Messaging</>
);

```